### DISCUSSION TITLE
Include in Email Notifications option for non-data fields

### DISCUSSION THREAD

**Discussion by RyanRoberts** [verbb/formie#1784] [Ideas] [answered]
Would you consider enabling this option for fields that don't contain data? Specifically the _Heading_ field type.

In my particular case I have a heading above the Captcha field and submission button, but neither of those are displayed in the notification emails so the heading is redundant in the email content.

![Google Chrome - 2024-03-27--11-38-43@2x](https://github.com/verbb/formie/assets/108874/1a33911f-b471-4dbc-8b0f-7ef69034cf5c)

I'm using the "All Form Fields" variable in the notification email, I guess it's only relevant when that's used.

Thanks!

**Comment by engram-design** [comment]
Sure, I was on the fence about whether or not the include cosmetic fields (field that have no actual content) in email notifications, or if they're more presentational for the front-end. Will add to the todo list.

**Comment by engram-design** [comment, marked as answer]
Added for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.7"`.

**Reply by RyanRoberts** [reply]
Amazing! Thanks.
